wsd/Storage.hpp

```diff
@@ -561,4 +513,47 @@ class LocalStorage : public StorageBase
     static std::atomic<unsigned> LastLocalStorageId;
 };
 
+/// Represents whether the underlying file is locked
+/// and with what token.
+struct LockContext
```

Comment:
But does it also actually lock the file/context?
Also what locking policy is being used? 
Like: One context per one thread at any given time? Or globally?

Reply:
This is to lock a document in storage. The policy is exclusive access while the document is open for editing.